### ISSUE TITLE
Materia orgánica o Carbono orgánico?

### ISSUE THREAD

**Issue by angelini75** [INTA-Suelos/SiSinta#95]
@dariorodriguez en la parte de los datos analíticos tenemos una seción llamada materia orgánica y una fila que se llama C%. El dato que se está cargando es el de Carbono organico, es así? Cuando exportas los datos a un archivo csv el campo (o columna) que corresponde a ese dato se llama analitico_materia_organica. Ahí @mauriciopasquier debería corregir o el nombre del campo en la base de datos o en la ficha de carga, según corresponda.

**Comment by dariorodriguez** [comment]
Cierto @angelini75 Los datos cargados corresponden al Carbono orgánico. Para sacar la materia orgánica habría que multiplicar ese valor por 1.724. Puede ser que ahí esté la confusión. La tabla de sisinta copia la ficha de reconocimiento y ahí está C Org

**Comment by angelini75** [comment]
Entonces @mauriciopasquier en la base de datos hay que reemplazar el nombre analisis_materia_organica por analisis_carbono_organico

**Comment by mauriciopasquier** [comment]
Lo cambio también en la [tabla de analíticos](http://sisinta.inta.gob.ar/perfiles/241/analiticos)?

**Comment by mauriciopasquier** [comment]
listo!

**Comment by dariorodriguez** [comment]
No. En la tabla de analíticos estaba bien. Es Materia Orgánica que incluye C %, N % y C/N

**Comment by mauriciopasquier** [comment]
Oh.. Pero en el CSV se exporta(ba)

- `analisis_materia_organica_c`
- `analisis_materia_organica_n`
- `analisis_materia_organica_cn`

Lo que me piden es que cambie el primero a `analisis_carbono_organico`, pero los demás dejarlos como están? Hice el cambio para todos porque los veo como campos relacionados (por la tabla de analíticos)

**Comment by angelini75** [comment]
@mauriciopasquier es cierto, la lógica que usaste es correcta. Sin embargo creo que se presta a confusión.
Propongo que se discuta el nombre de los campos de la tabla csv (o de la base de datos?) Qué tal en la wiki<https://github.com/INTA-Suelos/SiSinta/wiki/Documentaci%C3%B3n>

**Comment by mauriciopasquier** [comment]
El nombre de los campos en el csv lo saco de las columnas de la base de datos, es posible hacer un nombre específico pero me parece mejor mantener la consistencia.
